### PR TITLE
Gene overview page

### LINKED ISSUE #160: Create the new endpoint for coverage overview
Something like this: https://github.com/Clinical-Genomics/chanjo-report/blob/5660600f434c84f35f9142d412808202811aa1e4/chanjo_report/server/blueprints/report/views.py#L46

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Gene coverage overview page (fix #160), reachable when you click on any gene link on the geneS overview page:

image

<hr>

This link opens this page:

image
 

### How to test:
- Deploy this branch on cg-vm1 (already done by me)
- Go to demo genes overview page: https://chanjo2-stage.scilifelab.se/overview/demo
- Click on any gene link and see if the gene overview page is loaded

### Expected outcome:
- [x] The page should be working

### Review:
- [ ] Code approved by HS
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
This is ready for review now. Sonarcloud is flagging it because of repeated code in the HTML templates

image

I'm aware of this (see #195). it's a easy fix but I don't want to add too many code changes in just one PR. I plan to do it in another PR.

File: src/chanjo2/meta/handle_report_contents.py [hunk 1]
```diff
@@ -317,3 +320,76 @@ def get_genes_overview_incomplete_coverage_rows(
                     genes_overview_rows.append(overview_line)
 
     return _remove_none_elements(tuple_list=genes_overview_rows)
+
+
+#### Functions used to create a gene overview report ####
+
+
+def get_gene_overview_data(form_data: GeneReportForm, session: Session):
```
Comment (henrikstranneheim) [inline src/chanjo2/meta/handle_report_contents.py:328]:
```suggestion
def get_gene_overview_coverage_stats(form_data: GeneReportForm, session: Session):
```

**Review by henrikstranneheim — APPROVED** [review]
Nice 👍

**Comment by northwestwitch** [comment]
Merging! Thanks for the review @henrikstranneheim. I'll fix the code duplication thing in another PR.

**Pull request merged**